externs/olcsx.js

```diff
@@ -5,6 +5,45 @@
  */
 var olcsx;
 
+
+/**
+ * @typedef {{
+ *   map: (!ol.Map),
+ *   target: (Element|string|undefined),
+ *   createSynchronizers: (undefined|function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)
```

Comment:
I agree in the case of optional functions that putting undefined first is more readable.
However, it is not the way it is done in ol3. Do we want to add an exception to the coding style?